assembly/assembly.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+  <id>assembly</id>
+
+  <formats>
+    <format>zip</format>
+    <format>tar.gz</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
+  <dependencySets>
+    <dependencySet>
+      <outputDirectory>lib</outputDirectory>
+      <useProjectArtifact>false</useProjectArtifact>
+      <useTransitiveDependencies>false</useTransitiveDependencies>
+      <excludes>
+        <exclude>io.camunda:c7-data-migrator-distro</exclude>
+        <exclude>com.h2database:h2</exclude>
+      </excludes> 
+    </dependencySet>
+    <dependencySet>
+      <outputDirectory>userlib/</outputDirectory>
```

Comment:
/suggest Can we move the `userlib` folder into the `configuration` folder? Like this, the user can find everything they can control in one folder. This would also be consistent with Camunda Run, even though that's not a hard requirement. But I think it makes sense; otherwise, I would challenge why we would need a single file, `application.yml` in a dedicated `configuration` folder. This indirection wouldn't create much value. However, when including all user-owned, this folder makes much sense.

Reply:
Updated. Changes: https://github.com/camunda/c7-data-migrator/pull/15/commits/64a63a9d261c92cb1a51c0985db2611d7fcdf057